modules/nlu/src/backend/application/scoped/prediction-handler.ts

```diff
@@ -71,7 +71,7 @@ export class ScopedPredictionHandler {
       }
     }
 
-    if (this.isEmpty(nluResults) || this.isError(nluResults)) {
+    if (this.isEmpty(nluResults)) {
       throw new Error(`No model found for the following languages: ${languagesToTry}`)
```

Comment:
Can we add an error object with a message and stack trace to the understanding object?

Reply:
Sure, we could! Let's just do it in another PR